components/tyk-enterprise-portal/values.yaml

```diff
@@ -32,12 +32,31 @@ license: ""
 #In case you want to deploy enterprise portal as a standalone app, you can configure a different dashboard URL using
 #the parameter bellow
 overrideTykDashUrl: ""
-
+# Enterprise portal can be deployed as StatefulSet or as Deployment
+kind: StatefulSet
 storage:
+  # User can set the storage type for portal.
+  # Supported types: fs, s3, db
+  type: "fs"
   database:
     type: "sqlite3"
     connectionString: "db/portal.db"
+  # Configuration values for using s3 as storage for Tyk Enterprise Portal
+  # In case you want to provide the key ID and access key via secrets please
+  # refer to the existing secret inside the helmchart or the
+  # .Values.global.secrets.useSecretName variable
```

Comment:
One last thing. 
Can you define in the comment of `useSecretName` which keys need to be set in the secret?